app/views/providers/about.html.erb

```diff
@@ -1,7 +1,7 @@
-<%= content_for :page_title, "Edit 'About your organisation'" %>
+<%= content_for :page_title, "#{@errors.present? ? "Error: " : ""}Edit 'About your organisation'" %>
```

Comment:
I wonder if we can extract this so that `@errors` modified the title on any page? I've missed this feature out on the outcomes and entry requirements pages.

Reply:
We might be able to move this to the layout file instead but fine for this PR